src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const correctState = state;
+
+  for (const action of actions) {
```

Comment:
You should try destructuring of `action` here

Reply:
Denys removed this point from the checklist :)
![image](https://user-images.githubusercontent.com/105542865/190096864-b82f692b-abbb-450f-898b-9192790cfde7.png)